alphadia/libtransform.py

```diff
@@ -313,6 +319,11 @@ def forward(self, input: SpecLibBase) -> SpecLibBase:
         device = utils.get_torch_device(self.use_gpu)
 
         model_mgr = ModelManager(device=device)
+
+        if self.peptdeep_model_type is not None:
```

Comment:
mb add a comment here that this overwrites the `generic` model loaded in `ModelManager`